### PR TITLE
Improve feedback messages - Fixes #15 - Fixes #12

### LINKED ISSUE #12: Improve error messages feedback
Here there are some more error messages that should be explained: 

```
/tmp/tmp.20EQ4mStJB/SubmissionTest.java:8: error: cannot find symbol
  Transporte transporte;
  ^
  symbol:   class $(CLASS)
  location: class Viaje

Means: You have not defined the $(CLASS) class, enum or interface
```

```
/tmp/tmp.dpGQNuSEXp/SubmissionTest.java:17: error: cannot find symbol
    return transporte.costoTotal() + alojamiento.arancel();
                     ^
  symbol:   method $(METHOD)
  location: variable transporte of type Transporte

Means: a method $METHOD is missing
```

```
/tmp/tmp.wVYRZJ74TN/SubmissionTest.java:17: error: incompatible types: possible lossy conversion from double to int
    return transporte.costoTotal() + alojamiento.arancel();

Means: your are operating different number types (like int and double)
```

```
/tmp/tmp.D7c9Ajpysc/SubmissionTest.java:60: error: incompatible types: Micro cannot be converted to Transporte
  viaje = new Viaje(micro, departamento);
                    ^
Means: missing interface implementation or class inheritance, or simply incompatible types
```

```
/tmp/tmp.sx8rIxU7kJ/SubmissionTest.java:77: error: constructor Hotel in class Hotel cannot be applied to given types;
  Hotel hotel = new Hotel(4, 5);
                ^
  required: no arguments
  found: int,int
  reason: actual and formal argument lists differ in length
Means: Missing constructor or declared with different arities. 
```

```
/tmp/tmp.HOpovnhrxI/SubmissionTest.java:35: error: Bicicleta is not abstract and does not override abstract method costoTotal() in Transporte
class Bicicleta implements Transporte {
^
Means: missing interface method implementation
```

### LINKED ISSUE #15: Improvements to current feedback
See the following Spanish feedback:

* No se encontró la definición de class Golondrina en class SubmissionTest
  * It is not important where the class was declared - remove the _en class SubmissionTest_. 
* No se encontró la definición de variable anastasia en class Main
   * It should say _en Main_ o _en la clase Main_
* No se encontró la definición de method reanimarConUnChocolate() en variable golondrina of type Golondrina
   * _variable golondrina of type Golondrina_ should be also translated

### PR DESCRIPTION
**Pull request by rodri042**
Fixes #15
Fixes #12

### CODE REVIEW

File: spec/feedback_runner_spec.rb [hunk 1]
```diff
@@ -66,7 +67,56 @@ class Foo {
...
+      let(:request) {req(%q{
+    class Golondrina {
+    }}, %q{
+      public void testFoo(){
+        Golondrina golondrina = new Golondrina();
+        golondrina.reanimarConUnChocolate();
+      }
+    })}
+
+      it {expect(feedback).to include("* No se encontró la definición de el método `reanimarConUnChocolate()` en la variable `golondrina` de tipo `Golondrina`")}
```
Comment (flbulgarelli) [inline spec/feedback_runner_spec.rb]: del

Comment (rodri042) [reply spec/feedback_runner_spec.rb]: Hmm yes, but I think is not that easy. We've the same prolem with 'a'/'an' in english...

Comment (rodri042) [reply spec/feedback_runner_spec.rb]: Maybe we could use `la definición de `el método reanimarConUnChocolate()`?

**Comment by flbulgarelli** [comment]
Does this PR also fix https://github.com/mumuki/mumuki-java-runner/issues/12? It is very similar, but I think it has some extra messages.

**Comment by rodri042** [comment]
> Does this PR also fix https://github.com/mumuki/mumuki-java-runner/issues/12? It is very similar, but I think it has some extra messages. 

Not every of them, but I could add them to this PR

**Comment by rodri042** [comment]
screenshot1

:roll_eyes:

**Comment by flbulgarelli** [comment]
> Not every of them, but I could add them to this PR

Yes, please proceed! 

And don't mind about the code climate warning.

**Comment by rodri042** [comment]
> Yes, please proceed! 

@flbulgarelli Done!

**Pull request merged**